LICENSE.txt

```diff
@@ -1,27 +1,674 @@
-Copyright (c) 2015-2018, conda-forge
```

Comment:
Please do not modify this license file

Reply:
Thanks for your feedback, @synapticarbors. I had modified the license file to match the license of the Python package I intend to build for conda-forge. Is this not what the license file is supposed to reflect? At least that's how I understood the documentation.